numpy/lib/tests/test_io.py

```diff
@@ -1272,6 +1272,13 @@ def test_commented_header(self):
             assert_(w[0].category is np.VisibleDeprecationWarning)
         assert_equal(test, ctrl)
 
+    def test_names_and_comments_none(self):
+        #tests case when names is true but comments is None
```

Comment:
One final comment - it's nice to reference the github issue with `gh-10780` in test comments